crates/cubecl-linalg/src/matmul/components/global/multi_stage/double_buffering/base.rs

```diff
@@ -0,0 +1,14 @@
+#[derive(Clone)]
+pub enum BufferId {
+    A,
+    B,
+}
+
+impl BufferId {
+    pub fn to_u32(&self) -> u32 {
+        match self {
+            BufferId::A => 0,
+            BufferId::B => 1,
+        }
+    }
+}
```

Comment:
This could be extracted into a shared module in `matmul/components`. All of those things could reduce the compilation size at some point.